### PR TITLE
A Few More Disease and Reagents

### PR DESCRIPTION
**Pull request by Fox-McCloud**
A few more reagent diseases and such, again, very heavily based on Goon. Credit where credit is due.

- Adds in non-contagious GBS
 - Cure with Cryoxadone; obtained by injecting "gibbis" reagent into someone 
- Adds in Grave Fever
 - Does increasing tox damage and makes the infected person fall over from achy joints, also slows them down; vampire's diseased touch uses this
- Adds in Space Kuru
 - Makes you laugh uncontrollably, eventually kills you, obtained via prions

- Poison traitor bottles can now contain prions and gibbis
- Poison Traitor bottles can no longer be purchased by scientists or chemists: RD still can
- Brain burgers now contain prions instead of mannitol (don't eat people's brains--not good for ya, you know). Brain cakes are ok
- Initro now has a recipe (crank+bath salts+krokodil+nicotine+morphine+atropine+histamine)
- Adds in Pure Hugs and Pure Love
 - Whapping someone with a plushie has a chance to add pure hugs ot htem
 - Pure Love switches you off of harm intent and make you spam hug people in range
- Adds in ghost chilli juice and dragon's breath; botany oriented chems that are horrifically hot
- adds in space heart worms; they eat your heart (traitor poison bottles)
- Overhauled random drug bottles; now a much wider range of drugs

Code Wise
- Adds in a slowed var, which, if set, will cause you to move very slowly while not impacting other stats
- Random drugs and traitor poison bottles are now handled with json strings (big thanks to @Tastyfish for proper formatting of this!)

Fixes
- Fixes rejuv not clearing druggy

:cl: Fox McCloud
add: Adds in 3 disease: Grave Fever, Space Kuru, and Non-contagious GBS
add: Adds Reagent to cause Space Kuru and Non-contagious GBS to traitor poison bottles
add: Initropidril now has a recipe
add: Adds a number of new reagents
tweak: Brain burgers now contain prions instead of mannitol
tweak: Scientists and chemists can no longer purchase traitor poison bottles
tweak: random drug bottles now have a much wider range of reagents in them
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/human/human_movement.dm [hunk 1]
```diff
@@ -12,6 +12,8 @@
 	if(embedded_flag)
 		handle_embedded_objects() //Moving with objects stuck in you can cause bad times.
 
+	if(slowed)
```
Comment (tigercat2000) [inline code/modules/mob/living/carbon/human/human_movement.dm]: if you're making this at a living level you need to implement this in living/movement_delay() as well

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 2]
```diff
@@ -787,7 +787,7 @@
 	New()
 		..()
 		reagents.add_reagent("nutriment", 6)
-		reagents.add_reagent("mannitol", 6)
+		reagents.add_reagent("prions", 10)
```
Comment (KasparoVy) [inline code/modules/reagents/reagent_containers/food/snacks.dm]: Bahaha, brainburgers are going to be so great.

**Comment by QQCucumber** [comment]
Could do without the Something Awful memes

**Comment by Fox-McCloud** [comment]
>Could do without the Something Awful memes

I do hope you realize that GBS has been a thing for literally *years* on SS13 now.

File: code/datums/diseases/kuru.dm [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+		if(4)
+			if(prob(25))
+				to_chat(affected_mob, "<span class='danger'>You feel like you are going to die!</span>")
+				affected_mob.adjustOxyLoss(75)
+				affected_mob.adjustBruteLoss(75)
+				affected_mob.drop_l_hand()
+				affected_mob.drop_r_hand()
+				affected_mob.Stun(10)
+				affected_mob.Weaken(10)
+				affected_mob.visible_message("<span class='danger'>[affected_mob] laughs uncontrollably!</span>")
```
Comment (Crazylemon64) [inline code/datums/diseases/kuru.dm]: Oh boy I can't wait to RP the joker

**Comment by fludd12** [comment]
With extreme amounts of ghetto cryo and healing chems, it could be possible to survive Kuru forever.

I can't wait to try it.

**Comment by Bigtimetripod** [comment]
Woo! Thats what i ment Fox in one of the suggestion topics on forums. MOAR GOON DEATH CHEMS!

Initro having a recipe is good and all, but almost  impossible to make. Once tried to make it, but fell short on bathsalts. Mugwort is something that is near impossible to get... going thru the code and i think i remember that Pruple Dukes tea could be used to get some... but havent tried yet.

**Comment by monster860** [comment]
Prions? Wow. Just keep it accurate. I've seen enough fucked up prions in "Aurora".

**Comment by Fox-McCloud** [comment]
>Prions? Wow. Just keep it accurate. I've seen enough fucked up prions in "Aurora".

https://en.wikipedia.org/wiki/Prion

**Comment by Fox-McCloud** [comment]
Better random drugs

File: strings/chemistry_tools.json [hunk 4]
```diff
@@ -0,0 +1,473 @@
...
+		"suicider",
+		"moonshine",
+		"prions",
+		"grave_dust",
+		"nanomachines",
+		"chlorine",
+		"fluorine"
+	],
+	"CYBERPUNK_drug_adulterants_safe": [
+		"mercury",
```
Comment (Crazylemon64) [inline strings/chemistry_tools.json]: Ahh, mercury, the safest of adulterants

File: code/modules/mob/living/carbon/human/human_movement.dm (hunk 1 above)
Comment (Fox-McCloud) [inline code/modules/mob/living/carbon/human/human_movement.dm]: anddd done.

**Comment by fludd12** [comment]
I just realized something - this is all just an excuse to be able to say, "I'm making love in scichem" and be on paradise, rather than citadel.

**Pull request merged**